### DISCUSSION TITLE
Readd the late-bind setting

### DISCUSSION THREAD

**Discussion by eris-webserv** [PaperMC/Paper#10545] [Ideas]
### Is your feature request related to a problem?

Spigot used to have a configuration known as `late-bind`. It moved server binding to the end of plugin loading, a useful feature for proxies such as Bungeecord.
At some point, however, it was removed. It would be highly useful if this could be added back.

### Describe the solution you'd like.

Readd the patch that allows `late-bind` to be used, and add the configuration for it in paper.yml.

### Describe alternatives you've considered.

- Don't use late-bind
Without late-bind, some plugins would think the server on a proxy is online and atempt to connect. **This is bad, as it throttles requests.**

**Comment by eris-webserv** [comment]
It's been a month, any status on this?

**Comment by electronicboy** [comment]
low priority at best

**Comment by lynxplay** [comment]
You are always free to open a PR for this yourself :+1: 
The team currently simply has other priorities.

~~cat beat me to it~~

**Comment by eris-webserv** [comment]
I see, thank you for clarifying.
~~it's been ignored for a month so I thought I might as well ask~~
